Code/Framework/AzNetworking/AzNetworking/UdpTransport/UdpNetworkInterface.cpp

```diff
@@ -124,8 +124,15 @@ namespace AzNetworking
         }
     }
 
-    ConnectionId UdpNetworkInterface::Connect(const IpAddress& remoteAddress)
+    ConnectionId UdpNetworkInterface::Connect(const IpAddress& remoteAddress, uint16_t localPort)
     {
+        if (m_connectionSet.GetConnection(remoteAddress) != nullptr)
+        {
+            AZLOG_INFO("Attempting to connect to an endpoint (%s) that already has a connection", remoteAddress.GetString().c_str());
```

Comment:
Is INFO appropriate here? IMO this is WARN worthy.

Reply:
I could upgrade this.. ultimately it's just a timing issue in our API, of two hosts attempt to connect to each other at roughly this situation could arise, such as in a peer to peer mesh.